engine/consensus/approvals/assignment_collector.go

```diff
@@ -99,36 +101,10 @@ func (ac *AssignmentCollector) collectorByBlockID(incorporatedBlockID flow.Ident
 	return ac.collectors[incorporatedBlockID]
 }
 
-// authorizedVerifiersAtBlock pre-select all authorized Verifiers at the block that incorporates the result.
-// The method returns the set of all node IDs that:
-//   * are authorized members of the network at the given block and
-//   * have the Verification role and
-//   * have _positive_ weight and
-//   * are not ejected
-func (ac *AssignmentCollector) authorizedVerifiersAtBlock(blockID flow.Identifier) (map[flow.Identifier]*flow.Identity, error) {
```

Comment:
* I removed the pointer receiver, which allows to compute `authorizedVerifiersAtBlock`  before initializing the `AssignmentCollector`. 
* As this method is now not part of `AssignmentCollector`, I moved it to the end of the file